website/docs/getting-started/installation.mdx

```diff
@@ -0,0 +1,135 @@
+---
+title: Installing Taqueria
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<!-- -----
+Content Type: How-to Guide
+
+Purpose: To guide a developer through process of installing the Taqueria CLI, basic plugins, and the VS Code Extension 
+
+Audience: Novice to intermediate developers with a very basic understanding of Taqueria and CLI usage
+
+Goal: To provide a simple, easy, and reliable installation process for developers to follow so there is minimal churn due to frustration or inability to use the product due to installation or configuration issues
+
+Questions:
+- What do you need to understand about Taqueria to understand the installion process?
+- What options are there for installing Taqueria?
+- What are the exact steps a user will need to follow for each OS?
+- What are the most frequent friction points new users will have?
+
+----- -->
+
+## Taqueria Installation Basics
+
+Installing Taqueria is a quick and easy process if you understand a few basic concepts and follow the steps outlined in this guide
+
+Taqueria is a software suite that includies a CLI interface, a library of Node.js plugins, and a VS Code extension. Each of these are installed separatley. 
```

Comment:
agreed, given that the plugins are now installed as a taq command rather than being part of the initial installation we could also omit it entirely here IMO